global/meta/models.yml

```diff
@@ -112,6 +112,12 @@ organization:
     default: 0
     minimum: 0
 
+  vote_decrypt_public_main_key:
+    type: string
+    description: Public key from vote decrypt to validate cryptographic votes.
+    calculated: true
```

Comment:
What do you expect, this calculated-key is for?

Reply:
It contains the public main key from vote-decrypt so the clients can fetch it.